usb/usb_host_msc/src/msc_host_vfs.c

```diff
@@ -33,8 +33,14 @@ static esp_err_t msc_format_storage(size_t block_size, size_t allocation_size, c
 
     // Valid value of cluster size is between sector_size and 128 * sector_size.
     size_t cluster_size = MIN(MAX(allocation_size, block_size), 128 * block_size);
+
+#if ESP_IDF_VERSION <= ESP_IDF_VERSION_VAL(4, 4, 2)
```

Comment:
Since there will probably be a v4.4.3 with the same API as v4.4.2, suggest rephrasing this as:
```suggestion
#if ESP_IDF_VERSION < ESP_IDF_VERSION_VAL(5, 0, 0)
```

Reply:
Good point